rust/cubesql/cubesql/src/compile/snapshots/cubesql__compile__tests__quoted_keyword.snap

```diff
@@ -0,0 +1,9 @@
+---
+source: cubesql/src/compile/mod.rs
+expression: result
+---
++----------------+
+| quoted_keyword |
++----------------+
+| select         |
```

Comment:
PostgreSQL actually wraps keywords with quotes, and accepts keywords unwrapped only in assigning column alias position, but does not accept those in subquery alias or in column expression.

Maybe just wrap any `%I` argument with quotes? I can't imagine right now what could go wrong, in a sense that `format` argument can already be quoted in result, so any consumer should handle it as well. That would differ from PostgreSQL behavior, of course. But current implementation also differs, and "quote everything" approach have a benefit of something like "any %I substitution would make a proper SQL".